src/components/learner-credit-management/BudgetDetailAssignments.jsx

```diff
@@ -57,32 +58,53 @@ const BudgetDetailAssignments = ({
 
   return (
     <section className="budget-detail-assignments">
-      <h3 className="mb-3" ref={assignedHeadingRef}>
-        <FormattedMessage
-          id="lcm.budget.detail.page.assignments.heading"
-          defaultMessage="Assigned"
-          description="Heading for the assigned section on the budget detail page"
-        />
-      </h3>
-      <p className="small mb-4">
-        <FormattedMessage
-          id="lcm.budget.detail.page.assignments.description"
-          defaultMessage="Assigned activity earmarks funds in your budget so you can not overspend.
+      {isRetired ? (
+        <>
+          <h3 className="mb-3 text-info-900" ref={assignedHeadingRef}>
```

Comment:
[clarification] Where is the `text-info-900` coming from? I'm assuming you're finding an available Paragon color to try to mimic the changed text color in the Figma mock (i.e., the HEX value in Figma is not a valid Paragon color). 

However, note that this color change in Figma is not communicating that the text color itself should change, but rather just calling attention that, "hey, this particular messaging changed" using a lo-fi purple.

We should keep the color styles as they were out-of-the-box.